### PR TITLE
docs: refactor zh and rewrite connectors chapter

### PR DESCRIPTION
**Pull request by LenaLenaPan**

### CODE REVIEW

File: docs/zh_CN/guide/connector.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+对于需要自定义数据源或与特定第三方集成的场景，eKuiper 提供了基于插件的拓展源连接器：
+
+- [SQL 源](./sources/plugin/sql.md)：定期从关系数据库中拉取数据。
+- [视频源](./sources/plugin/video.md)：用于查询视频流。
+- [Random 源](./sources/plugin/random.md)：用于生成随机数据的源，用于测试。
+- [Zero MQ 源](./sources/plugin/zmq.md)：从 Zero MQ 读取数据。
+
+## 数据 Sink 连接器
+
+eKuiper 汇连接器负责将 eKuiper 处理后的数据发送到各种目标端点或系统，可直接与 MQTT、Neuron、EdgeX 等平台对接，并提供缓存机制以应对网络中断场景，确保数据的一致性。此外，用户还可通过动态属性和资源重用来定制接收行为，简化集成并提高可伸缩性。
```
Comment (ngjaying) [inline docs/zh_CN/guide/connector.md:42]: 汇 连接器是否更改

File: docs/zh_CN/guide/connector.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+与源连接器类似，Sink 连接器也分为内置和插件式两种。
+
+**内置汇连接器**
```
Comment (ngjaying) [inline docs/zh_CN/guide/connector.md:46]: The same, Maybe do a search

File: docs/zh_CN/guide/connector.md (hunk 1 above)
Comment (LenaLenaPan) [reply docs/zh_CN/guide/connector.md]: updated

File: docs/zh_CN/guide/connector.md (hunk 2 above)
Comment (LenaLenaPan) [reply docs/zh_CN/guide/connector.md]: updated

**Pull request merged**